docs/flow/modin/experimental/core/execution/ray/implementations/pandas_on_ray/index.rst

```diff
@@ -0,0 +1,20 @@
+:orphan:
+
+ExperimentalPandasOnRay Execution
+=================================
+
+`ExperimentalPandasOnRay` execution keeps the underlying mechanisms of :doc:`PandasOnRay </flow/modin/core/execution/ray/implementations/pandas_on_ray/index>`
+execution architecturally unchanged and adds experimental features of ``Data Transformation``, ``Data Ingress`` and ``Data Egress`` (e.g. `read_pickle_distributed`).
+
+PandasOnRay and ExperimentalPandasOnRay differences
+---------------------------------------------------
+- another Factory ``PandasOnRayFactory`` -> ``ExperimentalPandasOnRayFactory``
+- another IO class ``PandasOnRayIO`` -> ``ExperimentalPandasOnRayIO``
```

Comment:
Maybe we should describe one of the experimental features here (e.g. read_pickle_distributed)?

Reply:
We have a list of experimental functions with docstring descriptions. I don't think we need to duplicate information.

Reply:
Those docstrings say what the functions do. I meant to describe the logic of one of the functions, i.e. how does a function work but not what does a function do.

Reply:
It works with the appropriate combination of dispatcher and parser. If you are talking about this, then this is a general mechanism, duplication will turn out again.

Reply:
What duplication are you saying about? We do not have docs on experimental features, do we?

> It works with the appropriate combination of dispatcher and parser.

It relates to a concrete execution. I think we should add some description on how read/write from/to multiple files happen (probably, under `ExperimentalPandasOnRay Execution` section).

Reply:
Can you explain what needs to be added?

Ok, we have docstring for [read_pickle_distributed](https://modin--3819.org.readthedocs.build/en/3819/flow/modin/experimental/pandas.html#modin.experimental.pandas.read_pickle_distributed). What is missing here?

The interaction of classes is described [here](https://modin--3819.org.readthedocs.build/en/3819/flow/modin/core/execution/ray/implementations/pandas_on_ray/index.html#data-ingress), I am referring to it with amendments.

Reply:
For example, we read multiple files in parallel by parsing data in workers using experimental parser.

Reply:
Updated the docstring of `read_pickle_distributed` function a bit.

The first part of your statement follows from the docstring, and the second part from the "Data Ingress" diagram.

Reply:
Let's update docstrings of all the experimental features then.

Reply:
`to_pickle_distributed` docstring is updated as well. Others docstrings LGTM.

Reply:
ok